src/OpenProject.Revit/Data/RevitView.cs

```diff
@@ -15,149 +15,140 @@ public static class RevitView
     //Generate a VisualizationInfo of the current view
     //</summary>
     //<returns></returns>
-    public static BcfViewpointViewModel GenerateViewpoint(UIDocument uidoc)
+    public static BcfViewpointViewModel GenerateViewpoint(UIDocument uiDoc)
     {
       try
       {
-        var doc = uidoc.Document;
-
+        Document doc = uiDoc.Document;
         var bcfViewpoint = new BcfViewpointViewModel();
 
         //Corners of the active UI view
-        var topLeft = uidoc.GetOpenUIViews()[0].GetZoomCorners()[0];
-        var bottomRight = uidoc.GetOpenUIViews()[0].GetZoomCorners()[1];
+        XYZ bottomLeft = uiDoc.GetOpenUIViews()[0].GetZoomCorners()[0];
+        XYZ topRight = uiDoc.GetOpenUIViews()[0].GetZoomCorners()[1];
```

Comment:
One of the core fixes: The revit api returns on `GetZoomCorners` returns not top left and bottom right corner, but instead top right and bottom left.